Dockerfile

```diff
@@ -5,6 +5,9 @@ ADD . /src
 RUN make
 
 FROM docker.io/alpine:3.15
-RUN apk add -U ncurses-libs
+ARG TIMEZONE=UTC
 COPY --from=0 /src/tty-clock /usr/bin/tty-clock
 CMD ['/usr/bin/tty-clock']
+RUN apk add -U ncurses-libs tzdata && \
+    cp -rf /usr/share/zoneinfo/${TIMEZONE} /etc/localtime
```

Comment:
why isn't this just a TZ environment that can be passed by the `docker run` caller?

Reply:
ah, that works too, you can set the default for the container with the build argument, as well as override it at runtime with the environment `TZ`. I could remove the default, but I think its nice to be able to set it and not have to specify it again.